### PR TITLE
Fix lots of lint errors.

### PR DESCRIPTION
**Pull request by dlorenc**
# Changes

These were found in github.com/tektoncd/pipeline/issues/794. This PR
fixes most, but not all of those errors.

# Submitter Checklist

N/A

### CODE REVIEW

File: cmd/entrypoint/main.go [hunk 1]
```diff
@@ -30,7 +30,6 @@ import (
 
 var (
 	ep       = flag.String("entrypoint", "", "Original specified entrypoint to execute")
-	waitFile = flag.String("wait_file", "", "If specified, file to wait for")
```
Comment (vdemeester) [inline cmd/entrypoint/main.go]: This is definitely needed

Comment (dlorenc) [reply cmd/entrypoint/main.go]: D'oh, not sure why it looked unused.

**Comment by bobcatfish** [comment]
This linter seems to be a bit off base :thinking:

**Comment by bobcatfish** [comment]
@dlorenc do you want to keep going ahead with this or close it for now?

**Comment by dlorenc** [comment]
> @dlorenc do you want to keep going ahead with this or close it for now?

Yeah it should be good to go now. Someone started using one of the unused variables so there was a merge conflict.

**Comment by afrittoli** [comment]
This commit made third_party/VENDOR-LICENSE empty - was that on purpose?

**Comment by dlorenc** [comment]
> This commit made third_party/VENDOR-LICENSE empty - was that on purpose?

Definitely not :(

**Pull request merged**